File: Backend/dependency/notifications/src/main/java/com/staffinghub/coding/challenges/dependency/notifications/EmailHandler.java
package com.staffinghub.coding.challenges.dependency.notifications;

import com.staffinghub.coding.challenges.dependency.inquiry.Inquiry;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.stereotype.Component;

@Component
public class EmailHandler {

    private static final Logger LOG = LoggerFactory.getLogger(EmailHandler.class);

    public void sendEmail(final Inquiry inquiry) {
        LOG.info("Sending email for: {}", inquiry);
    }

}

File: Backend/dependency-kotlin/notifications/src/main/kotlin/com/staffinghub/coding/challenges/dependency/notifications/EmailHandler.kt
package com.staffinghub.coding.challenges.dependency.notifications

import com.staffinghub.coding.challenges.dependency.inquiry.Inquiry
import mu.KotlinLogging
import org.springframework.stereotype.Component

private val logger = KotlinLogging.logger {}

@Component
class EmailHandler {
    fun sendEmail(inquiry: Inquiry) {
        logger.info {
            "Sending email for: $inquiry"
        }
    }
}
